Recipes/Recipes.DataAccess/Entities/Relationships/TagRecipeEntity.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace Recipes.DataAccess.Entities.Relationships
+{
+    public class TagRecipeEntity
+    {
+        public int RecipeId { get; set; }
```

Comment:
I assume these cannot be null in DB? Why not make them required? Also, maybe use default instead of null?

Reply:
Are we talking about a [required] attribute or the required keyword? The keyword throws a compilation error if a nullable property is set to null. In this case int is already not nullable so there is no need for it. Not sure about the attribute, but I think it serves the same purpose.